docs/setup/production.asciidoc

```diff
@@ -144,6 +145,13 @@ bin/kibana -c config/instance1.yml
 bin/kibana -c config/instance2.yml
 --------
 
+[float]
+[[accessing-load-balanced-kibana]]
+=== Accessing multiple load-balanced Kibana clusters
+To be able to access multiple load-balanced Kibana clusters from the same browser, the setting `xpack.security.cookieName` must be set in the configuration to avoid having conflicts between cookies from the different Kibana instances.
```

Comment:
```suggestion
To be able to access multiple load-balanced {kib} clusters from the same browser, the setting `xpack.security.cookieName` must be set in the configuration to avoid having conflicts between cookies from the different {kib} instances.
```